core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/io/image.kt

```diff
@@ -0,0 +1,60 @@
+package org.jetbrains.kotlinx.dataframe.impl.io
+
+import java.awt.Graphics2D
+import java.awt.RenderingHints
+import java.awt.image.BufferedImage
+import java.awt.image.ImageObserver
+import java.io.ByteArrayOutputStream
+import javax.imageio.ImageIO
+import kotlin.math.max
+import kotlin.math.min
+
+internal fun BufferedImage.resizeKeepingAspectRatio(
```

Comment:
Is this a new implementation, or maybe from kotlin dl?

Reply:
No, it’s not from Kotlindl; it’s simply a wrapper around the Java API)